lib/md054.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+
+"use strict";
+
+const { addErrorContext } = require("../helpers");
+const {
+  filterByTypes,
+  filterByPredicate,
+  getTokenTextByType
+} = require("../helpers/micromark.cjs");
+
+const isInlineLink = ({ children }) => children.some(
+  ({ type }) => type === "resource"
+);
+
+const deepGetTokenTextByType = (tokens, type) => getTokenTextByType(
+  filterByTypes(tokens, [ type ]),
+  type
+);
+
+const fixInfo = (tokens, link) => {
+  if (isInlineLink(link)) {
+    return null;
+  }
+  const label = deepGetTokenTextByType([ link ], "labelText");
+  const definitions = filterByTypes(tokens, [ "definition" ]);
+
+  const definition = filterByPredicate(
+    definitions,
+    (d) => deepGetTokenTextByType([ d ], "definitionLabelString") === label
+  );
+
+  if (definition.length > 0) {
+    const destination = deepGetTokenTextByType(
+      definition,
+      "definitionDestination"
+    );
+    return {
+      "editColumn": link.endColumn,
+      "insertText": `(${destination})`
```

Comment:
Is there ever a need to escape this?

Reply:
I was asking something different. Could destination be "text)text" in which case that ")" would terminate the chunk early? Maybe in that case we need to insert "(text\)text)". Or maybe if there's a quote or a space in it?